server/hl7/hl7.controller.js

```diff
@@ -117,6 +117,24 @@ function getUserFiles(req, res, next) {
   return next(err);
 }
 
+/**
+ * Get get a file given it's id
+ * @returns {file}
+ */
+function getFile(req, res, next) {
+  const currentUser = req.user;
+  const { fileId } = req.params
+  const file = currentUser.files.find((f) => {
+    return f._id === fileId
+  })
+
+  if (file) {
+    return res.status(httpStatus.OK).json(file)
+  }
+  const err = new APIError('The requested file does not exist', httpStatus.NO_CONTENT);
```

Comment:
```suggestion
  const err = new APIError('The requested file does not exist', httpStatus.NOT_FOUND);
```

[204 No Content](https://http.cat/204) is a success message (I've only seen normally used for successful DELETEs). We want an error message that says the file doesn't exist, so [404 Not Found](https://http.cat/404) seems more appropriate.

Reply:
+1